openhtf/exe/triggers.py

```diff
@@ -42,31 +34,21 @@
 
 _LOG = logging.getLogger(__name__)
 
-def AutoStart(serial='UNKNOWN_DUT_ID'):  # pylint: disable=invalid-name
-  """Start the test immediately with the given serial (or a dummy DUT ID)."""
-  return serial
-
 
-def AutoStop(dummy_dut_id):  # pylint: disable=invalid-name
-  """Stop the test immediately regardless of DUT ID given."""
-  pass
+def AutoStart(serial='UNKNOWN_DUT_ID'):  # pylint: disable=invalid-name
+  """DEPRECATED, use a lambda instead."""
+  # TODO(madsci): Remove this after transitionary period.  Note it was broken
+  # before (it should have returned lambda: serial), so I doubt anyone is
+  # actually using it, but just to be safe.
+  raise AttributeError(
+      'DEPRECATED, use lambda: "foo" instead of AutoStart("Foo")')
 
 
 def PromptForTestStart(message='Provide a DUT ID in order to start the test.',
-                       text_input=False, timeout_s=60*60*24):
+                       text_input=True, timeout_s=60*60*24):
   """Make a test start trigger based on prompting the user for input."""
   def trigger():  # pylint: disable=missing-docstring
     prompt_manager = user_input.get_prompt_manager()
     return prompt_manager.DisplayPrompt(
         message, text_input=text_input, timeout_s=timeout_s)
   return trigger
-
-
-def PromptForTestStop(message='Hit ENTER to complete the test.',
-                      text_input=False, timeout_s=60*60*24):
-  """Make a test stop trigger based on prompting the user for a response."""
-  def trigger(dummy_dut_id):  # pylint: disable=missing-docstring
-    prompt_manager = user_input.get_prompt_manager()
-    return prompt_manager.DisplayPrompt(
-        message, text_input=text_input, timeout_s=timeout_s)
-  return trigger
```

Comment:
ah, think we should leave it for now until we have a better solution for 'recent tests', then?